### PR TITLE
Issue 60 no content and content

### LINKED ISSUE #60: Icon doesnt update when notification appears while page is open
Also the div#emptycontent is still present which looks strange
screen shot 2016-02-04 at 16 22 41

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #60 

The error from #60 happens, when you perform an action on a notification which does not succeed. The counter between notifications and the available notifications in the UI are then off, so you can have notifications, although the counter is 0 which shows the empty content message additionally to the content.

Other commits are styling fixes.

@PVince81 @tomneedham

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -369,7 +364,7 @@
 		 * Returns how many notifications in the UI
 		 */
 		numNotifications: function() {
-			return OCA.Notifications.num;
+			return _.keys(this.notifications).length;
```
Comment (PVince81) [inline js/app.js]: clever

**Comment by PVince81** [comment]
:+1:

File: js/app.js (hunk 1 above)
Comment (tomneedham) [inline js/app.js]: Yup, that was the clever way to do it :)

**Comment by tomneedham** [comment]
:+1:

**Pull request merged**